### PR TITLE
Numero instagram

### PR DESCRIPTION
**Pull request by guillaumemoby**
Before sending your pull-request 💌 make sure :

- [x] The tests are up-to-date and your code is tested

### CODE REVIEW

**Comment by guillaumemoby** [comment]
We have a problem on ios but not on Android:
android: 
Screenshot_2020-11-16-09-21-43-256_host exp exponent
ios:

File: src/Components/drills/FrisbeeDrillIllustration.js [hunk 1]
```diff
@@ -88,14 +89,26 @@ const FrisbeeDrillIllustration = (props) => {
...
+  };
+
+  const renderPagination = (index) => {
     return (
       <Pagination
         dotsLength={props.drill.steps.length}
-        activeDotIndex={activeIndex}
+        activeDotIndex={index}
         containerStyle={{
-          backgroundColor: theme.BACKGROUND_COLOR_LIGHT,
```
Comment (sallesma) [inline src/Components/drills/FrisbeeDrillIllustration.js:97]: I think your bug came from this line

It should be better with a transparent background

File: src/Components/drills/FrisbeeDrillIllustration.js [hunk 2]
```diff
@@ -88,14 +89,26 @@ const FrisbeeDrillIllustration = (props) => {
...
+      );
+    }
+  };
+
+  const renderPagination = (index) => {
     return (
       <Pagination
         dotsLength={props.drill.steps.length}
-        activeDotIndex={activeIndex}
+        activeDotIndex={index}
```
Comment (sallesma) [inline src/Components/drills/FrisbeeDrillIllustration.js:108]: This breaks the effect used on line 21 : when you move to another drill in the top navigation, it should display the first step. If you use the index directly, it stays on the step it was displaying

**Comment by guillaumemoby** [comment]
It's good on Iphone

**Review by sallesma — APPROVED** [review]

**Pull request merged**